config/urls.py

```diff
@@ -13,7 +13,8 @@
     1. Import the include() function: from django.urls import include, path
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
-from django.urls import path
+from django.urls import path, include
```

Comment:
```suggestion
from django.urls import (
    path, 
    include
)
```
한 줄에서 import 하고자 하는 대상이 두 개 이상일 경우 개별 줄로 분리해주는 것이 위코드 코딩 컨벤션입니다 :-)

Reply:
넹